models/users.ts

```diff
@@ -0,0 +1,33 @@
+import { object, string, number, array, TypeOf } from "yup";
+
+export const requestUserSchema = object({
+  id: string().ensure().required(),
+  first_name: string().ensure().required(),
+  last_name: string().ensure().required(),
```

Comment:
I know you copied this from another branch, but I am probably going to ask them to change these models to camelCase instead of snake_case. I would change them over here for right now, and then see where the application breaks, fix them, so you dont have to stress too much about merge conflicts.